src/main/java/mat/client/util/FeatureFlagConstant.java

```diff
@@ -0,0 +1,15 @@
+package mat.client.util;
+
+public class FeatureFlagConstant {
+
+    public final static String MAT_ON_FHIR ="MAT_ON_FHIR";
```

Comment:
btw,
can we use an interface for constants? it would less verbose then.
but, I don't know if our code convention allows interfaces for contants

Reply:
I used Interface for Constants